authapi/tests/test_organizations.py

```diff
@@ -767,8 +676,196 @@ def test_create_permission_for_organizations_team(self):
+        resp = self.client.post(
+            reverse(
+                'seedorganization-teams-permissions-list',
+                args=[org2.pk, team2.pk]
+            ),
+            data={
+                'type': 'org:admin',
+                'object_id': org1.pk,
+                'namespace': 'foo'
+            })
+        self.assertEqual(resp.status_code, status.HTTP_403_FORBIDDEN)
+
+        # Wrong object id
+        resp = self.client.post(
+            reverse(
+                'seedorganization-teams-permissions-list',
+                args=[org1.pk, team1.pk]
+            ),
+            data={
+                'type': 'org:admin',
+                'object_id': org2.pk,
+                'namespace': 'foo'
+            })
+        self.assertEqual(resp.status_code, status.HTTP_403_FORBIDDEN)
+
+        # Correct
+        resp = self.client.post(
+            reverse(
+                'seedorganization-teams-permissions-list',
+                args=[org1.pk, team1.pk]
+            ),
+            data={
+                'type': 'org:admin',
+                'object_id': org1.pk,
+                'namespace': 'foo'
+            })
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # team:admin
+        resp = self.client.post(
+            reverse(
+                'seedorganization-teams-permissions-list',
+                args=[org1.pk, team1.pk]
+            ),
+            data={
+                'type': 'team:admin',
+                'object_id': team1.pk,
+                'namespace': 'foo'
+            })
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # other
+        resp = self.client.post(
+            reverse(
+                'seedorganization-teams-permissions-list',
+                args=[org1.pk, team1.pk]
+            ),
+            data={
+                'type': 'foo:bar',
+                'object_id': '7',
+                'namespace': 'foo'
+            })
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+    def test_permission_create_permission_team_admin(self):
+        '''Users with team:admin permissions should be able to give other teams
+        team:admin, or any other permission except org:admin.'''
+        user, token = self.create_user()
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+        org = SeedOrganization.objects.create(title='test org')
+        team = SeedTeam.objects.create(title='test team', organization=org)
+        self.add_permission(user, 'team:admin', team.pk)
+
+        # team:admin
+        resp = self.client.post(
+            reverse(
+                'seedorganization-teams-permissions-list',
+                args=[org.pk, team.pk]
+            ),
+            data={
+                'type': 'team:admin',
+                'object_id': team.pk,
+                'namespace': 'foo'
+            })
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
```

Comment:
Added in https://github.com/praekelt/seed-auth-api/pull/28/commits/d5ed595301ce8af210714ef27c89efe7c4d9a472